src/ui/abstract/ScriptObject.mjs

```diff
@@ -0,0 +1,6 @@
+import BaseScriptObject from '../../script/Object';
+
+class ScriptObject extends BaseScriptObject {
```

Comment:
What does `BaseScriptObject` give us? Why not just localize it all to `ScriptObject` here?

Reply:
Good point, it's purely to avoid having a local identifier naming clash between:

- `script/Object` (official client's `FrameScriptObject`)
- `ui/abstract/ScriptObject` (official client's `CScriptObject`)

Perhaps we could find more appropriate names for both or one of them?

Reply:
We should call `script/Object` `FrameScriptObject`, I think (a la the client).

Reply:
Done! 👍